app/models/account_summary.rb

```diff
@@ -19,7 +19,7 @@ class AccountSummary < ApplicationRecord
   scope :filtered, -> { where.missing(:follow_recommendation_suppressions) }
 
   def self.refresh
-    Scenic.database.refresh_materialized_view(table_name, concurrently: false, cascade: false)
+    Scenic.database.refresh_materialized_view(table_name, concurrently: true, cascade: false)
```

Comment:
I wonder if something like the following could avoid the migration changes and possible edge cases:
```suggestion
    populated = ActiveRecord::Base.connection.execute(ActiveRecord::Base.sanitize_sql_array(['SELECT relispopulated FROM pg_class WHERE relname = :table_name', table_name: self.table_name])).values.first&.first
    Scenic.database.refresh_materialized_view(table_name, concurrently: !populated, cascade: false)
```

Reply:
Would a simple query against the view not work? I'm not sure this is really needed

Reply:
I don't think there's anything in `schema.rb` that makes the views be created with no data when doing `db:schema:load`.

Reply:
> Would a simple query against the view not work? I'm not sure this is really needed

What do you mean? A query against the view would not populate it, but error out afaik. But I guess instead of this we could try something like:

```suggestion
    begin
      Scenic.database.refresh_materialized_view(table_name, concurrently: true, cascade: false)
    rescue ActiveRecord::StatementInvalid
      Scenic.database.refresh_materialized_view(table_name, concurrently: false, cascade: false)
    end
```

Reply:
Correct -- the edge case only shows up after a `bin/rails db:migrate`, not a `schema:load` (the `no_data` impacts that migration itself, but not the resulting schema, so if you load from schema they do get populated in a way where you can subsequently run with concurrent true) ... so I guess that's another solution/expectation that further reduces as well -- anyone setting up a new instance who set up the db from a schema load and not a full migration run would be fine.

Reply:
I think that `rescue` idea is a cleaner approach both in here and in the linked PR base class thing I was trying.

Reply:
The issue wouldn't arise from a brand new install, and it would be very uncommon otherwise; cases where it could arise are basically running the migrations and then not running the code from before this PR. This could happen if a person stops the services, run migrations in steps checking out intermediary releases, and then restart the services.

That's why I would like a fallback instead of or in addition to past migration changes. The `rescue` approach is probably the best because it won't run anything more in the common and happy case, and will not perform black PostgreSQL magic to find out the unhappy case.